examples/secagg-mt-pytorch/run_clients.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Number of client processes to start
+N=5 # Replace with your desired value
+
+# Start N client processes
+for i in $(seq 1 $N)
+do
+  nohup python client.py &
```

Comment:
Is `nohup` required here?

Reply:
no, not really. just wanted to hide output from `client.py`. I've removed nohup